website/pages/_meta.ts

```diff
@@ -1,29 +1,8 @@
 const meta = {
-  index: '',
-  '-- 1': {
-    type: 'separator',
-    title: 'GraphQL.JS Tutorial',
+  docs: {
+    type: 'page',
+    title: 'Documentation'
```

Comment:
we need to group mdx pages under some folder and assert `type: page` to it, so we'll have proper sidebar